internal/task/queue.go

```diff
@@ -130,7 +130,7 @@ func (t *Queue) worker() {
 			klog.ErrorS(err, "requeuing", "key", item.Key)
 			t.queue.AddRateLimited(Element{
 				Key:       item.Key,
-				Timestamp: time.Now().UnixNano(),
+				Timestamp: 0,
```

Comment:
Why not change the timestamp to seconds, instead of removing it altogether? I would think a timestamp is needed for ratelimiting?

Reply:
Changing it to seconds will only make the ratelimit work as expected in the first second. And still cause some elements never destroy when time > 1s